Sprint-2/implement/tally.js

```diff
@@ -1,3 +1,17 @@
-function tally() {}
-
+function tally(list) {
+    let count = {};
+    if(list.length == 0){
+        return {};
+    }
```

Comment:
Do you actually need this special-case of an early return? If you delete this code, will things break, or still work the same?